README.md

```diff
@@ -79,10 +78,10 @@ spec:
    address: "threescale-istio-adapter:3333"
 ```
 
-Create the required resources:
+Create the required resources in the namespace where the Istio `ControlPlane` exists:
```

Comment:
this looks like it could use some more clarification - maybe saying istio-system would be the most likely namespace but you might have multitenancy so it could be different, or just add a command to list istio pods in all namespaces to find the right one?